src/lib/markbind/src/parser.js

```diff
@@ -305,14 +305,12 @@ Parser.prototype._preprocess = function (node, context, config) {
     delete element.attribs.inline;
     delete element.attribs.trim;
 
-    fileContent = isTrim ? fileContent.trim() : fileContent;
-
     if (includeSrc.hash) {
       // directly get segment from the src
       const segmentSrc = cheerio.parseHTML(fileContent, true);
       const $ = cheerio.load(segmentSrc);
       const htmlContent = $(includeSrc.hash).html();
```

Comment:
Could we call this variable `fragmentContent` instead? I think trying to mentally differentiate between `htmlContent` and `fileContent` is confusing.